### PR TITLE
[Android] Output error message and stack trace in chat

### PR DESCRIPTION
**Pull request by cyx-6**
This PR enables the output of error message and stack trace in chat, when the chat backend fails.

### CODE REVIEW

File: android/MLCChat/app/src/main/assets/app-config.json [hunk 1]
```diff
@@ -9,4 +9,4 @@
     }
   ],
   "add_model_samples": []
-}
+}
```
Comment (Hzfengsy) [inline android/MLCChat/app/src/main/assets/app-config.json:12]: nit

File: android/prepare_libs.sh [hunk 2]
```diff
@@ -23,8 +23,8 @@ cmake .. \
       -DUSE_HEXAGON_SDK=OFF \
       -DMLC_LLM_INSTALL_STATIC_LIB=ON \
       -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON \
-      -DUSE_OPENCL=ON
+      -DUSE_OPENCL=ON \
+      -DUSE_CUSTOM_LOGGING=ON \
 
-make tvm4j_runtime_packed -j8
+make tvm4j_runtime_packed -j12
```
Comment (Hzfengsy) [inline android/prepare_libs.sh:29]: Would be great to use `${nproc}` here

**Review by Hzfengsy — APPROVED** [review]
General LGTM

File: android/MLCChat/app/src/main/assets/app-config.json (hunk 1 above)
Comment (cyx-6) [reply android/MLCChat/app/src/main/assets/app-config.json]: sure, fixed

File: android/prepare_libs.sh (hunk 2 above)
Comment (cyx-6) [reply android/prepare_libs.sh]: sure, added

**Pull request merged**